inbox.php

```diff
@@ -20,26 +20,77 @@
 <html>
     <head>
         <?php require_once('universal.inc') ?>
+        <link rel="stylesheet" href="css/event.css" type="text/css" />
         <link rel="stylesheet" href="css/messages.css"></link>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.13.18/css/bootstrap-select.min.css">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.13.18/js/bootstrap-select.min.js"></script>
+        <link rel="stylesheet" href="css/inbox.css">
         <script src="js/messages.js"></script>
+        <?php if ($accessLevel >= 2) : ?>
+            <script src="js/event.js"></script>
+        <?php endif ?>
         <title>Downtown Greens | Inbox</title>
     </head>
     <body>
+        <?php if ($accessLevel >= 2) : ?>
```

Comment:
Can we confirm that access level is still necessary

Reply:
I will have to discuss with others in the group if we are considering changing the access levels, or ask which access levels are assigned to which roles (admin, manager, etc.). It might not be necessary

Reply:
Got it. Let's discuss at the next meeting